research/research.go

```diff
@@ -0,0 +1,69 @@
+// nolint
+package main
```

Comment:
Update README please, add this folder instead of examples

Reply:
`dkc uses herumi/bls-eth-go-binary. The pseudocode of using herumi/bls-eth-go-binary in dkc is available here`

Reply:
Done!